tests/sparsezoo/helpers.py

```diff
@@ -23,11 +23,20 @@
 from sparsezoo.utils import CACHE_DIR
 
 
-def download_and_verify(model: str, other_args: Optional[Dict] = None):
+def download_and_verify(
+    model: str, test_name: Optional[str] = "", other_args: Optional[Dict] = None
```

Comment:
Suggestion: Remove `Optional` here, `Optional` implies `None` can be passed as the `test_name`, and `str(None)` will be equal to `"None"` on line 33 
```suggestion
    model: str, test_name: str = "", other_args: Optional[Dict] = None
```